portfolio/src/main/webapp/bandGen.js

```diff
@@ -5,20 +5,28 @@ function getMood() {
 }
 
 function generateBand() {
-    const NUMBER_OF_BANDS = 2;
+    const numberOfBands = 5;
     let mood = getMood();
     let bands = getBands();
     let bandContainer = document.getElementById("bandContainer");
-    let rand = Math.floor(Math.random() * NUMBER_OF_BANDS);
-    console.log(bands);
+    let rand = Math.floor(Math.random() * numberOfBands);
     bandContainer.innerHTML = bands[mood][rand];
 }
 
 function getBands() {
     let bands = {};
-    bands["Chill"] = ["Elijah Nang", "Joji"];
-    bands["Funky"] = ["Marc Rebillet", "TWRP"];
-    bands["Ecstatic"] = ["Big Bad Voodoo Daddy", "Taraf de Haidouks"];
-    bands["Fancy"] = ["Devil Doll", "The Speakeasies Swing Band"];
+    bands["Chill"] = ["Elijah Nang", "Joji", "Syros", "DopeSmoker",
+    "Mystic Sons"];
```

Comment:
Indent this line more to make it clear that it is the continuation of the previous line.
Same for all the other line breaks that follow